modules/navier_stokes/doc/content/syntax/Modules/NavierStokesFV/index.md

```diff
@@ -0,0 +1,105 @@
+# NavierStokesFV System
+
+## Overview
+
+The NavierStokesFV system is dedicated to decrease the effort required by the user to
+prepare simulations which need to solve the Navier Stokes equations. The action
+is capable of setting up:
+
+- +Incompressible+ and +weakly-compressible+ simulations for
+- +Clean fluids+ and flows in +porous media+
```

Comment:
I've been calling non-obstructed fluid flows clean fluid flows because there is nothing else in the domain besides the fluid. I can replace it with unobstructed or similar if you think it is better.

Reply:
I truly don’t know what s the right word here

Reply:
I'll keep the clean fluid now, I used it in 2-3 papers and readers/reviewers seemingly got the idea. If we see questions, I'll change it in the documentation here.